python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -527,8 +524,7 @@ def _bulk_updateStateTime(jobStateDict, bulk_time_lookup={} ):
         """
         for this_state, these_jobs in jobStateDict.iteritems():
             if bulk_time_lookup == {} or this_state not in bulk_time_lookup:
-                bulk_result = execute("getBulkStateTime(%s,\'%s\')" %
-                                        (repr([j.backend.id for j in these_jobs]), this_state))
+                bulk_result = execute("getBulkStateTime(%s,\'%s\')" % (repr([j.backend.id for j in these_jobs]), this_state))
```

Comment:
This should throw actually looking at the code. All a try/except would do here is forward the exception up the stack